src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ThemeContextProvider, { ThemeContext } from "./contexts/ThemeContext";
+import AppContextProvider from "./contexts/AppContext";
+import "./resets.css";
+import MainApp from "./MainApp";
+
+import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
Suggestion: I think it would be better to put imports for packages at the top, then relative imports at the bottom 😁